repository/SmalltalkCI-Coverage-Core.package/SCICoverallsWriter.class/class/coverallsBuildData.st

```diff
@@ -0,0 +1,12 @@
+coveralls
+coverallsBuildData
+
+	SmalltalkCI readOnlyFileNamed: self coverallsBuildDataFilename
+		do: [ :stream | 
+			[^ STON fromStream: stream] 
+				on: STONReaderError
+				do: [:error | 
+					SmalltalkCI failAndQuit: 
+						'Unable to parse Coveralls build data from ', self coverallsBuildDataFilename, ':', Character cr,
+						error asString, Character cr,
+						'This is a smalltalkCI internal error. If the problem persists, please open an issue at https://github.com/hpi-swa/smalltalkCI/issues/new .'. ]]
```

Comment:
Maybe we should move this line into `failAndQuit:` and rename the method to `quitWithInternalFailure:` or something like that. But we can change this later.